docs/4.3/architecture/teleport_auth.md

```diff
@@ -224,9 +224,9 @@ back-ends as shown in the table below:
 
 Data Type        | Supported Back-ends       | Notes
 -----------------|---------------------------|---------
-Cluster state    | `dir`, `etcd`, `dynamodb` | Multi-server (HA) configuration is only supported using `etcd` and `dynamodb` back-ends.
-Audit Log Events | `dir`, `dynamodb`         | If `dynamodb` is used for the audit log events, `s3` back-end **must** be used for the recorded sessions.
-Recorded Sessions| `dir`, `s3`               | `s3` is mandatory if `dynamodb` is used for the audit log.
+Cluster state    | `dir`, `etcd`, `dynamodb`,`firestore` | Multi-server (HA) configuration is only supported using `etcd`, `dynamodb`, and `firestore` back-ends.
+Audit Log Events | `dir`, `dynamodb`, `firestore`         | If `dynamodb` is used for the audit log events, `s3` back-end **must** be used for the recorded sessions.
+Recorded Sessions| `dir`, `s3`, `gcp`               | `s3` is mandatory if `dynamodb` is used for the audit log.
```

Comment:
I wonder whether it's mandatory to use `gcp` as storage if you have `firestore` as the backend, similar to how you must use `s3` if you use `dynamodb`.

@benarent @russjones Do you know?

Reply:
would be good to give guidance if we expect you need to have a s3 or google storage with firestore.

Reply:
Agreed. I'm not 100% sure why it's a hard requirement for AWS either.